gnd/src/main/java/com/google/android/gnd/repository/local/FeatureEntity.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.repository.local;
+
+import androidx.annotation.NonNull;
+import androidx.room.ColumnInfo;
+import androidx.room.Embedded;
+import androidx.room.Entity;
+import androidx.room.Index;
+import androidx.room.PrimaryKey;
+import com.google.auto.value.AutoValue;
+import com.google.auto.value.AutoValue.CopyAnnotations;
+
+/** Representation of a {@link com.google.android.gnd.vo.Feature} in local db. */
+@AutoValue
+@Entity(
+    tableName = "feature",
+    indices = {@Index("id")})
+public abstract class FeatureEntity {
+  /**
+   * Key used in {@link com.google.android.gnd.repository.local.FeatureEditEntity} when edit
```

Comment:
Where in the `FeatureEditEntity` would this appear?

Reply:
This will be actually defined in the LocalDataService class that will map between JSONObjects fetched from the DB and actual columns in Feature (i.e., location). Deleting for now.